srv/modules/runners/ui_rgw.py

```diff
@@ -119,7 +120,7 @@ def _urls(self):
                 self.credentials['urls'].append("http{}://{}".format(ssl, resource))
 
 
-def credentials(canned=None):
+def credentials(canned=None, **kwargs):
```

Comment:
When calling the runner from salt-api rest interface, salt-api adds an additional parameter. See #255